dashboard/src/actions/apps.test.tsx

```diff
@@ -8,6 +8,8 @@ import Chart from "../shared/Chart";
 import { definedNamespaces } from "../shared/Namespace";
 import { IAppState, UnprocessableEntity } from "../shared/types";
 
+// TODO: do we need to mock the actual modules - https://stackoverflow.com/a/54781786 Check.
+
```

Comment:
I was at this point when I stopped: I wanted to try testing this file, but was no longer able to test since `react-scripts-ts` no longer worked. I needed to update the mocks below to satisfy the type checking, but we may also need to add
```
jest.mock("../shared/App");
jest.mock("../shared/Chart");
```
here as well.